core/templates/dev/head/pages/exploration-player-page/layout-directives/progress-nav.directive.ts

```diff
@@ -103,6 +105,22 @@ angular.module('oppia').directive('progressNav', [
             }
           };
 
+          var doesInteractionHaveSpecialCaseForMobile = function() {
+            // The submit button should be shown:
+            // 1. In mobile mode, if the current interaction is either
+            //    ItemSelectionInput or MultipleChoiceInput.
+            // 2. In desktop/mobile, mode if ItemSelectionInput interaction
```

Comment:
`In desktop/mobile, mode if ItemSelectionInput interaction` -> `In desktop mode, if the current interaction is ItemSelectionInput` since the case with mobile is already handled in the previous point.

Also, the reason why I want to be so precise with the comment is so that it doesn't differ from the actually written code, because that can cause confusion with the developers and then they cannot be sure if the comment is wrong or the actual code is wrong.

Reply:
Done.